src/test/java/teammates/test/cases/ui/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -462,9 +462,9 @@ private void testDeleteCourseAction() {
         assertNotNull(BackDoor.getCourseAsJson(courseId));
         
         ______TS("delete course then proceed");
-        
+
         InstructorCoursesPage coursePage = courseEditPage.clickDeleteCourseLinkAndConfirm();
-        coursePage.verifyContains("Add New Course");
+        assertTrue(coursePage.getStatus().contains(String.format(Const.StatusMessages.COURSE_DELETED, courseId)));
```

Comment:
I think the coursePage.getStatus() call after deleting the course currently contains a status stating that the specified course is being deleted and a status stating that there are no courses created yet. This is because the one and only course is being deleted. If I were to use assertEquals will it cause any error in future tests when the course that is being deleted is not the only course available and therefore the status stating that there are no courses created yet won't be shown?